src/lora/lorawan_adr.erl

```diff
@@ -291,19 +291,13 @@
 -spec new(Region :: atom()) -> handle().
 new(Region) ->
     %% Filter gotthardp's table down to only 125kHz uplink DataRates.
-    FilterMapFn = fun
-        ({_, _, down}) ->
-            false;
-        ({DataRate, {Spreading, 125 = Bandwidth}, _Direction}) ->
-            {true, {DataRate, {Spreading, Bandwidth}}};
-        (_) ->
-            false
-    end,
-    Datarates = lists:filtermap(FilterMapFn, lorawan_mac_region:datars(Region)),
-    %% min-max refer to #device.min_datarate docs
-    [{MinDataRate, {MaxSpreading, _}} | _] = Datarates,
-    {MaxDataRate, {MinSpreading, _}} = lists:last(Datarates),
-    TxPowers = lorawan_mac_region:uplink_power_table(Region),
+    MinSF = min_adr_sf(Region),
+    MaxSF = max_adr_sf(Region),
+    _MinDataRate = min_adr_dr(Region),
+    MaxDataRate = max_adr_dr(Region),
+    Datarates = [{1, {7, 125}}],
```

Comment:
If these values work for all the ADR plans we support right now, can we move it into a macro?